frontend/src/js/controller.mainController.js

```diff
@@ -19,5 +21,16 @@
             localThis.msg = $cookies.getObject("msg");
             $cookies.remove("msg");
         }
+
+        function transferAccount(valid) {
+            if (!valid) {
+                return;
+            }
+            $http.post("/api/users/elixir_transfer",
+                    $.param({"_xsrf": $cookies.get("_xsrf")})
+                ).then(function(data) {
+                    window.location.replace( data.data.redirect );
+                });
+        }
```

Comment:
You should really create a new controller for this. Put it in `loginController` and use that on the HTML-page. Maybe we should rename this controller into `navbarController` in the future, because I think that is the only job it's doing.

Maybe the `transferAccount` post thingy should be moved into the `User` factory.